frontend/app/components/wp-edit-form/work-package-edit-form.ts

```diff
@@ -142,14 +142,14 @@ export class WorkPackageEditForm {
     this.closeAllEditFields();
 
     this.workPackage.save()
-      .then(() => {
+      .then(savedWorkPackage => {
         _.each(this.activeFields, (handler:WorkPackageEditFieldHandler) => {
           handler.deactivate();
         });
-        deferred.resolve(this.workPackage);
+        deferred.resolve(savedWorkPackage);
 
-        this.wpNotificationsService.showSave(this.workPackage, isInitial);
-        this.editContext.onSaved(this.workPackage);
+        this.wpNotificationsService.showSave(savedWorkPackage, isInitial);
+        this.editContext.onSaved(savedWorkPackage);
```

Comment:
@oliverguenther it makes sense to me to uniformly use the newly created work package here but maybe there are some side effects that I am not aware of. Can you please take an extra look here?